opentracing-spring-web-autoconfigure/src/test/java/io/opentracing/contrib/spring/web/autoconfig/ServerTracingAutoConfigurationTest.java

```diff
@@ -42,14 +50,48 @@ public MockTracer tracer() {
     @Autowired
     private MockTracer mockTracer;
 
+    @Autowired
+    @Qualifier("tracingFilter")
+    private FilterRegistrationBean tracingFilter;
+
+    @MockBean
+    private ServletFilterSpanDecorator mockDecorator1;
+    @MockBean
+    private ServletFilterSpanDecorator mockDecorator2;
```

Comment:
@ledor473 @jfbreault can you explain why there are these two decorators? 

Is it the decorator used by the filter? Then there is only one.

Reply:
There's 2 Decorators to ensure that the ObjectProvider can correctly provide more than 1 Decorator to the Filter. 

It's used by this test https://github.com/opentracing-contrib/java-spring-web/pull/62/files/09692c02b88b35acb4801ffe7e7ea86a8f49164d#diff-86e82c5211f5ddaf9510408e7ea71310R86

Reply:
Oh I see it's providing a bean for the auto-configuration. sorry